lib/asyncWork.dart

```diff
@@ -0,0 +1,152 @@
+import 'dart:async';
+
+void fetchNameDelayedTwoSeconds() async {
+  String name = await fetchName();
+  print('Мене звати $name');
+}
+
+Future<String> fetchName() async {
+  // Імітація затримки в 2 секунди
+  return await Future.delayed(Duration(seconds: 2), () {
+    return 'Олександр'; // Твоє ім'я
+  });
+}
+
+void fetchAgeDelayedMilliseconds() async {
+  String ageDigit = await fetchAge();
+  String yearWord = formatYearWord(ageDigit);
+  print('Мені $ageDigit $yearWord');
+}
+
+String formatYearWord(String age) {
+  int lastDigit = int.parse(age.substring(1));
```

Comment:
```suggestion
  final lastDigit = int.parse(age.substring(1));
```

**Type inference works here**, so there’s no need to explicitly declare the type.

See Dart documentation on type inference:

https://dart.dev/language/type-system#type-inference